_config.yml

```diff
@@ -31,7 +31,7 @@ exclude:
   - "vendor"
 # Build settings
 markdown: kramdown
-kramwdown:
+kramdown:
```

Comment:
The [kramdown config rule](https://github.com/howard-e/wai-website/pull/1/files#diff-ecec67b0e1d7e17a83587c6d27b6baaaa133f42482b07bd3685c77f34b62d883) was [misspelled](https://jekyllrb.com/docs/configuration/markdown/#kramdown-processor) so I suppose wasn’t technically being applied.

Other changes to support this have already been applied https://github.com/w3c/wai-tutorials/pull/783